### DISCUSSION TITLE
Add CheckboxGroup

### DISCUSSION THREAD

**Discussion by Jonas-C** [chakra-ui/zag#1584] [Ideas]
Hi! I'm currently wiring together my own version of a CheckboxGroup using a version of Chakras FormControl. It works, but is slightly cumbersome and error-prone when comparing it to things like the `RadioGroup` found in zag. Would it be possible to get this into zag as a potential wrapper for several checkbox machines?

Prior art: 
- [CheckboxGroup (chakra)](https://v2.chakra-ui.com/docs/components/checkbox#checkboxgroup)
- [CheckboxGroup (react aria)](https://react-spectrum.adobe.com/react-aria/CheckboxGroup.html)

**Comment by segunadebayo** [comment]
We discussed this yesterday with the team. The agreement was to add this to Ark, not Zag.js since it's not really state machine-related.

It's now implemented here: https://github.com/chakra-ui/ark/commit/9d2b7c57c11f677d6f978e7bdd9fc069adfc1878

It should be released shortly.

**Reply by Jonas-C** [reply]
You guys move quickly, thanks!
